tests/integration/_conftest/protostar_fixture_factory.py

```diff
@@ -47,6 +47,9 @@ def fake_activity_indicator(message: str) -> Generator[None, None, None]:
     yield
 
 
+REPOSITORY_ROOT = (Path(__file__).parent / ".." / ".." / "..").resolve()
```

Comment:
+10000
Please use parent wherever possible, if we use '..' and path goes through non-existing directory it will cause an error (once spent whole day debugging it)